src/components/email/Email.tsx

```diff
@@ -148,25 +140,6 @@ export default class Email extends React.Component<
                 {this.emailData.recipients.join(", ")}
               </div>
 
-              {this.emailData.cc ? (
-                <div className="recipients">
-                  <b>CC:</b>
-                  <span
-                    className="copyToClipboard"
-                    onClick={() =>
-                      this.handleClipboardCopy(
-                        { ccCopied: true },
-                        this.emailData.cc.join(", ")
-                      )
-                    }
-                  >
-                    {this.state.ccCopied ? "✅(copied)" : "🔗"}
-                  </span>
-                  &nbsp;
-                  {this.emailData.cc.join(", ")}
-                </div>
-              ) : undefined}
```

Comment:
thinking about this more - we could support cc's but only mash them together if clicking on mailto: link.